thrift/src/main/thrift/entity.thrift

```diff
@@ -0,0 +1,26 @@
+namespace java com.gu.contententity.thrift.entity
+
+include "entities/person.thrift"
+include "entities/film.thrift"
+include "entities/game.thrift"
+include "entities/restaurant.thrift"
+
+enum EntityType {
+  PERSON = 0
+  FILM = 1
+  GAME = 2
+  RESTAURANT = 3
+}
+
+// A container for an entity
+struct Entity {
+  1: required string id         //Unique entity ID, e.g. person/theresa_may/12345
+  2: required EntityType entityType
+  3: optional string tagId      //Guardian Tag ID
```

Comment:
it means that when I update a `tag` to add a reference to an entity, there should be no update on the `entity` instance, otherwise you need to keep the bidirectional relationship in sync which could end being very complicated.